CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+Contributing
+============
+
+This is a fork of `git@github.com:cucumber-ltd/shouty.java.git`
+
+In order to maintain updates from cucumber-ltd, and to support contributing back, we use GitFlow
+
```

Comment:
I haven't tried GitFlow in specific, but I've worked with various branching models that are similar. What I've found is that branching can pretty much be eliminated by changing development practices. 

One approach is to do everything on master, using feature toggles to disable work in progress.
GitFlow is the opposite. Use many branches and integrate relatively infrequently (compared to the master approach where integration happens for every commit).

For a small project like this I think the master approach is fine for most changes. For big changes (I don't see that happening often in this project) we can use feature branches and pull requests. I see no need to add a complex process like GitFlow yet. Maybe when there is a 12 person team working around the clock...